src/layout/datemetadata/metadata-datetime.js

```diff
@@ -0,0 +1,30 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * 
+ */
+const DateMetadata = ({ _body, datePublished , dateModified, timePublished, timeModified}) => (
```

Comment:
I think we can make this component more reusable. It should probably be called DescriptionList. And take in an array of objects.

Then we need to map over that and generate description lists accordingly.

Reply:
@sukhrajghuman that's be done and is committed now